music21/stream/tools.py

```diff
@@ -23,7 +23,7 @@
 # ------------------------------------------------------------------------------
 
 def removeDuplicates(thisStream: stream.Stream,
-                     classesToRemove: list | None = None,
+                     classesToRemove: list = [meter.TimeSignature, key.KeySignature, clef.Clef],
```

Comment:
Agree that these shouldn't be mutable: so you could make this immutable with a tuple